packages/terra-form/src/Input.jsx

```diff
@@ -55,7 +55,9 @@ class Input extends React.Component {
       ...customProps
     } = this.props;
     const additionalInputProps = Object.assign({}, customProps);
-    const inputClasses = cx(['input', additionalInputProps.className]);
+    const inputClasses = cx([
+      { input: (customProps.type !== 'radio' && customProps.type !== 'checkbox') },
+      additionalInputProps.className]);
```

Comment:
I would get a few more opinions on this before making any changes. This feels really awkward to me that we use this Input for Control but ignore all styles related to the Input component. Essentially just trying to tie into the couple extra lines of default / value and get the `<input>` tag. It feels like a really roundabout way of inheritance which isn't recommended in React. https://facebook.github.io/react/docs/composition-vs-inheritance.html

My opinion is that Input is an independent functioning component from Control and Control should use its' own input tag as Alex suggest on the tech design.

## Input
![image](https://user-images.githubusercontent.com/6720991/29976029-43ef6082-8efe-11e7-827a-c123ab80307d.png)

## Control
![image](https://user-images.githubusercontent.com/6720991/29976050-559fa972-8efe-11e7-800c-9168cd7ab5a1.png)

Reply:
I agree that this seems awkward. And it will probably become more so as we add OCS styles and special functionality for different types. I think that the control implementing it's own input tag and value logic would be fine, but I think another valid option would be for the Input to not have any styling at all and having HOCs that wrap it and add any styling/logic that's type specific.